src/pgduckdb_types.cpp

```diff
@@ -896,6 +897,7 @@ ConvertPostgresToBaseDuckColumnType(Form_pg_attribute &attribute) {
 		return duckdb::LogicalTypeId::UUID;
 	case JSONOID:
 	case JSONARRAYOID:
+	case JSONBOID:
```

Comment:
We should also support jsonb insida a postgres array:

```suggestion
	case JSONBOID:
	case JSONBARRAYOID:
```

This would need some tests in array_type_support.sql

Reply:
Done